util/majordomo.go

```diff
@@ -56,6 +57,16 @@ func InitMajordomo(ctx context.Context) (majordomo.Service, error) {
 		return nil, errors.Wrap(err, "failed to register file confidant")
 	}
 
+	httpConfidant, err := httpconfidant.New(ctx,
+		httpconfidant.WithLogLevel(LogLevel("majordomo.confidants.http")),
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create file confidant")
```

Comment:
Please could you change this error message to http confidant?  Thanks.

Reply:
Fix. Sorry, I overlooked that copy&paste error.

Reply:
Thanks!